.github/workflows/ci.yml

```diff
@@ -275,3 +275,64 @@ jobs:
         with:
           name: cypress-screenshots
           path: tests/cypress/screenshots
+
+  playwright:
+    # Do not run scheduled tests on tier repositories
+    if: github.repository == 'glpi-project/glpi' || github.event_name != 'schedule'
+    name: "Playwright tests using latest PHP and MariaDB versions"
+    runs-on: "ubuntu-latest"
+    timeout-minutes: 60
+    env:
+      COMPOSE_FILE: ".github/actions/docker-compose-app-e2e.yml:.github/actions/docker-compose-services.yml"
+      APPLICATION_ROOT: "${{ github.workspace }}"
+      DB_IMAGE: "githubactions-mariadb:10.11"
+      PHP_IMAGE: "githubactions-php-apache:8.3"
+      UPDATE_FILES_ACL: true
+    steps:
+      - name: "Set env"
+        run: |
+          echo "APP_CONTAINER_HOME=${{ runner.temp }}/app_home" >> $GITHUB_ENV
+      - name: "Checkout"
+        uses: "actions/checkout@v4"
+      - name: "Restore dependencies cache"
+        uses: actions/cache@v4
+        with:
+          path: |
+            ${{ env.APP_CONTAINER_HOME }}/.composer/cache/
+            ${{ env.APP_CONTAINER_HOME }}/.npm/_cacache/
+          key: "app_home_deps-8.3-${{ hashFiles('composer.lock', 'package-lock.json') }}"
+          restore-keys: |
+            app_home_deps-8.3-
+            app_home_deps-
+      - name: "Initialize containers"
+        run: |
+          .github/actions/init_containers-start.sh
+      - name: "Show versions"
+        run: |
+          .github/actions/init_show-versions.sh
+      - name: "Build GLPI's dependencies and translations"
+        run: |
+          docker compose exec -T app .github/actions/init_build.sh
+      - name: Install Playwright dependencies
+        run: docker compose exec -T app npm ci
```

Comment:
It should not be necessary as the result is expected to be similar as a regular `npm install` execution.